addons/web/static/src/core/file_input/file_input.js

```diff
@@ -30,6 +30,7 @@ export class FileInput extends Component {
 
         onMounted(() => {
             if (this.props.autoOpen) {
+                this.props.state.fileInput = this.fileInputRef.el;
```

Comment:
Not a good idea, you should never edit props. Only the owner of the state can edit is state. It s not a good pattern to do this. If you want to edit a props, you should pass a callback props to do it.

Reply:
Oh indeed 😵‍💫